.codecov.yml

```diff
@@ -2,7 +2,7 @@ coverage:
   status:
     project:
       default:
-        target: 75%
+        target: 50%
```

Comment:
👀

Reply:
can put this back up now... it's at 78% coverage